### ISSUE TITLE
about LoFTR result on mp3d

### ISSUE THREAD

**Issue by IceTTTb** [samiragarwala/PlaneFormers#5]
Hey, @samiragarwala @jinlinyi 
Great work!
I tried to evaluate the camera poses with LoFTR on the mp3d dataset. However, I only achieved 'rot(mean)=24.2; rot(<30)=78%' which was much worse than the result as shown in Tab 2. I used the weight of 'indoor_ds_new.ckpt' for LoFTR inference. Did I lose some detailed settings?

**Comment by samiragarwala** [comment]
Hi! Thanks for the interest in our work and the question. The results in the paper were reported after retraining LoFTR on the mp3d training set. After doing that, you should be able to get similar results on the test set. When we had run LoFTR using released checkpoints, we got a mean rotation error of 33.11 degrees and (% <= 30 degrees) of 67.06%. It may be helpful to check that you have set  _default_cfg['coarse']['temp_bug_fix'] = True before evaluating when using that checkpoint. Let us know if you have any follow-up questions!

**Comment by IceTTTb** [comment]
Thanks for your reply！ It is very helpful to me!
